src/components/GetStarted/EditPreferences.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+	<div class="edit-preferences">
+		<h2 class="edit-preferences__header">
+			Looking for different loans?
+		</h2>
+		<p class="edit-preferences__sub-header">
+			Change your preferences and find more borrowers to support.
+		</p>
+		<div class="row">
+			<div class="small-12 large-6 columns">
+				<!-- Causes Card -->
+				<div class="edit-preferences__card" v-if="causes.length > 0">
+					<div class="edit-preferences__icon-wrapper">
+						<span class="edit-preferences__icon-background">
+							<kv-icon
+								class="edit-preferences__check-icon"
+								name="checkmark"
+							/>
+						</span>
+					</div>
+					<div class="edit-preferences__content-wrapper">
+						<div class="row collapse text-left align-middle">
+							<div class="small-6 large-7 xlarge-6 columns">
+								<h2 class="edit-preferences__content-wrapper--title">
```

Comment:
Minor, but in BEM-land the '--' in a classname indicates a "modifier" class. So this would be saying you have a special version of the element, "content-wrapper" (that maybe you want to style differently than the normal content-wrapper).

A single hyphen would be a way to keep regular "element" status, like "content-wrapper-title". Or maybe just "content-title".

http://getbem.com/naming/
http://getbem.com/faq/#css-nested-elements

Reply:
👍